### ISSUE TITLE
can't run on webpack

### ISSUE THREAD

**Issue by sibelius** [tngan/samlify#312]
**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.

**Thank you in advance for helping us to improve this library!** Please read through the template below and answer all relevant questions. Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/). Finally, to avoid duplicates, please search existing Issues before submitting one here.

By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

I can't use this package when compiling my backend using webpack

### Reproduction

Add this package to this boilerplate https://github.com/entria/entria-fullstack/tree/master/packages/server
```jsx
Error: ENOENT: no such file or directory, open '/templates/encrypted-key.tpl.xml'
    at Object.openSync (fs.js:439:3)
    at Object.readFileSync (fs.js:344:35)
    at Object.eval (webpack-internal:///../../node_modules/xml-encryption/lib/utils.js:6:23)
    at eval (webpack-internal:///../../node_modules/xml-encryption/lib/utils.js:28:30)
```

### Environment

> Please provide the following:

- **Version of this library used:** latest
- **Version of the platform or framework used, if applicable:** node
- **Other relevant versions (language, server software, OS, browser):** mac, javascript, node
- **Other modules/plugins/libraries that might be involved:**

**Comment by sibelius** [comment]
related to https://github.com/auth0/node-xml-encryption/issues/55

**Comment by tngan** [comment]
@sibelius Did you get rid of it? Example [react-samlify](https://github.com/authenio/react-samlify) uses webpack as well.

**Comment by sibelius** [comment]
fixed adding this to my webpack config

```jsx
node: {
    fs: true,
    __dirname: true,
  },
```

tks

**Comment by tngan** [comment]
Good to hear. :)

**Comment by shellscape** [comment]
@tngan apologies for raising an old issue here, but this was been resolved in v1.0.0 of @authenio/xml-encryption. see the follow up on https://github.com/auth0/node-xml-encryption/issues/55
